doc/modules/gaussian_process.rst

```diff
@@ -94,6 +94,7 @@ the API of standard scikit-learn estimators, :class:`GaussianProcessRegressor`:
 * :ref:`sphx_glr_auto_examples_gaussian_process_plot_gpr_noisy.py`
 * :ref:`sphx_glr_auto_examples_gaussian_process_plot_compare_gpr_krr.py`
 * :ref:`sphx_glr_auto_examples_gaussian_process_plot_gpr_co2.py`
+* :ref:`sphx_glr_auto_examples_classification_plot_classification_probability.py`
```

Comment:
In this example the `GaussianProcessClassifier` is used (not the `GaussianProcessRegressor`). Therefore, this example should be linked under the classification section. There is the section "1.7.3. GPC examples" that explains some examples in detail. However, I think it's best for now to just link this example at the end of section "1.7.2. Gaussian Process Classification (GPC)". Could you make a new examples section there and place the link under it? (Compare this and the following lines for how to do it: https://github.com/scikit-learn/scikit-learn/blob/main/doc/modules/gaussian_process.rst?plain=1#L91)

Reply:
This is now done.

Cannot believe I missed that Classifier/Regressor difference.

Thank you.